components/timezone/src/provider.rs

```diff
@@ -21,6 +21,11 @@ use tinystr::TinyAsciiStr;
 use zerovec::ule::{AsULE, ULE};
 use zerovec::{ZeroMap2d, ZeroSlice, ZeroVec};
 
+pub mod names;
```

Comment:
issue: not sure if we can say this fixes https://github.com/unicode-org/icu4x/issues/2909 since this is just the data model, there's no fetcher struct yet

Reply:
Just pushed the API